Keyboard and Controller Improvements

MechaDragonX opened the pull request:
Based on yifan --> master merge.

MechaDragonX committed:
Controller Input Coded Using XInput.NET v2017.04-2

XInput.net is from https://github.com/speps/XInputDotNet.

MechaDragonX committed:
Implemented Controller Support and Adjusted Jump code

Adjusted Jump code was applied to the Keyboard script as well.
The camera moves up down as the player jumps. This should be fized.

MechaDragonX committed:
Attempt to Fix Keyboard Jump Code

No cap to jump height.  You can jump multiple consecutive times without proper timing.
XInput's fine.

Commit:
Revert "Minor Changes"

This reverts commit 250bfac32580df3e71e227348c10cbe5cd7dffff.

Conflicts:
	MasterProj/Assembly-CSharp.csproj
	MasterProj/Assets/testScene.unity
	MasterProj/Library/CurrentLayout.dwlt
	MasterProj/Library/metadata/04/048e7aea81d03164ca438289d306a906
	MasterProj/Library/metadata/04/048e7aea81d03164ca438289d306a906.info
	MasterProj/Library/metadata/07/071be1250a9669a4da56fd18a833d0ac
	MasterProj/Library/metadata/07/071be1250a9669a4da56fd18a833d0ac.info
	MasterProj/Library/metadata/07/07b4b79622286ff4eb0d4f8ed1315728.info
	MasterProj/Library/metadata/09/095b8e6eda8038b4ab4dbe21848a4b95
	MasterProj/Library/metadata/09/095b8e6eda8038b4ab4dbe21848a4b95.info
	MasterProj/Library/metadata/0d/0db2571a427a97f4da522e322936699a
	MasterProj/Library/metadata/0d/0db2571a427a97f4da522e322936699a.info
	MasterProj/Library/metadata/10/10145f279f90346bbb61712ff585828b
	MasterProj/Library/metadata/12/127afd5151fb9c84aba21294cc58ed36
	MasterProj/Library/metadata/12/127afd5151fb9c84aba21294cc58ed36.info
	MasterProj/Library/metadata/13/130fdf994343e5b4da22dacf23f7c5fb
	MasterProj/Library/metadata/13/130fdf994343e5b4da22dacf23f7c5fb.info
	MasterProj/Library/metadata/13/13780d262bc481141813c1e3b2bb8c84.info
	MasterProj/Library/metadata/16/16950083e19ff214aa7059393b02d716
	MasterProj/Library/metadata/16/16950083e19ff214aa7059393b02d716.info
	MasterProj/Library/metadata/16/16f3d63ddb9e71947b60f0f150be0e74
	MasterProj/Library/metadata/16/16f3d63ddb9e71947b60f0f150be0e74.info
	MasterProj/Library/metadata/17/17f49fc376290ad4a873de4c01a87ce9
	MasterProj/Library/metadata/17/17f49fc376290ad4a873de4c01a87ce9.info
	MasterProj/Library/metadata/18/18364813e9887f74ebbd42f53a70b0f5
	MasterProj/Library/metadata/18/18364813e9887f74ebbd42f53a70b0f5.info
	MasterProj/Library/metadata/18/1855ff962e31a7f4b995f31e3fceb065
	MasterProj/Library/metadata/18/1855ff962e31a7f4b995f31e3fceb065.info
	MasterProj/Library/metadata/19/190a2a2d0507d7641ab939cae19ef64c
	MasterProj/Library/metadata/19/190a2a2d0507d7641ab939cae19ef64c.info
	MasterProj/Library/metadata/1a/1aff40e145d6e1f4caa3e94aeb432df9
	MasterProj/Library/metadata/1a/1aff40e145d6e1f4caa3e94aeb432df9.info
	MasterProj/Library/metadata/1b/1b223e448a4e18c489a8e8db6fe79799
	MasterProj/Library/metadata/1b/1b223e448a4e18c489a8e8db6fe79799.info
	MasterProj/Library/metadata/1c/1cd8858de87b6954394b87b77b4c9b0b
	MasterProj/Library/metadata/1c/1cd8858de87b6954394b87b77b4c9b0b.info
	MasterProj/Library/metadata/1c/1ceb5b1434074954d9d1f906d7ee2278
	MasterProj/Library/metadata/1c/1ceb5b1434074954d9d1f906d7ee2278.info
	MasterProj/Library/metadata/1d/1de44e7bf68e0a64ab55311ddc2ca2a6
	MasterProj/Library/metadata/1d/1de44e7bf68e0a64ab55311ddc2ca2a6.info
	MasterProj/Library/metadata/21/2184969c41bccd1449334573d65ea12a
	MasterProj/Library/metadata/21/2184969c41bccd1449334573d65ea12a.info
	MasterProj/Library/metadata/24/2454229bf98dd2241b5fbb1d4c3e176d
	MasterProj/Library/metadata/24/2454229bf98dd2241b5fbb1d4c3e176d.info
	MasterProj/Library/metadata/24/24b43c20b29042d40b103f2c87f4fea4
	MasterProj/Library/metadata/24/24b43c20b29042d40b103f2c87f4fea4.info
	MasterProj/Library/metadata/25/25ebe415cc14c4f4ea15c34a796b75bb
	MasterProj/Library/metadata/26/2691ab1c87b7f7749a4b799180bead4b
	MasterProj/Library/metadata/26/2691ab1c87b7f7749a4b799180bead4b.info
	MasterProj/Library/metadata/27/27d60ba6f2ee100428390816358c8e05.info
	MasterProj/Library/metadata/2a/2a297c7ca09cc144e83f30a5958ef7ab
	MasterProj/Library/metadata/2b/2b9c4907e052ff549963f4f6f2e0710d
	MasterProj/Library/metadata/2b/2b9c4907e052ff549963f4f6f2e0710d.info
	MasterProj/Library/metadata/2d/2de72026c3d433b4dae873365322ccf8
	MasterProj/Library/metadata/2d/2de72026c3d433b4dae873365322ccf8.info
	MasterProj/Library/metadata/2e/2eb2d0388ed974b3cba47458803bd275
	MasterProj/Library/metadata/2f/2f19c37188a75ec4db9a6483db1ddd96
	MasterProj/Library/metadata/2f/2f19c37188a75ec4db9a6483db1ddd96.info
	MasterProj/Library/metadata/30/30ec0e41d6a065742a3e283d7a8070e6
	MasterProj/Library/metadata/30/30ec0e41d6a065742a3e283d7a8070e6.info
	MasterProj/Library/metadata/3b/3b57bc223f534004a88bc504e1b2c734
	MasterProj/Library/metadata/3b/3b57bc223f534004a88bc504e1b2c734.info
	MasterProj/Library/metadata/3b/3b7977825b011d44d8da0c32fc18a708
	MasterProj/Library/metadata/3b/3b7977825b011d44d8da0c32fc18a708.info
	MasterProj/Library/metadata/3b/3bab3506592bbbe468e6d7eb1088a198
	MasterProj/Library/metadata/3b/3bab3506592bbbe468e6d7eb1088a198.info
	MasterProj/Library/metadata/3e/3e508200404409544a0db2fa0256ea84.info
	MasterProj/Library/metadata/3e/3ea2078dd8d4391489ad68586bcbe1c9
	MasterProj/Library/metadata/3e/3ea2078dd8d4391489ad68586bcbe1c9.info
	MasterProj/Library/metadata/40/40e904e85e740fd4c867f4d27406b0cd
	MasterProj/Library/metadata/40/40e904e85e740fd4c867f4d27406b0cd.info
	MasterProj/Library/metadata/41/4112618c310f6a049b24f6c9b0873b5f
	MasterProj/Library/metadata/41/4112618c310f6a049b24f6c9b0873b5f.info
	MasterProj/Library/metadata/46/465954785e078164189e12219d06c4df
	MasterProj/Library/metadata/46/465954785e078164189e12219d06c4df.info
	MasterProj/Library/metadata/46/46c02d8d6499b4309ac1ec5d0b818585
	MasterProj/Library/metadata/49/49d0a0c630486ae40a7af16b2c61aa21
	MasterProj/Library/metadata/49/49d0a0c630486ae40a7af16b2c61aa21.info
	MasterProj/Library/metadata/49/49d187ecef3e8a8478dba4c373bed2db
	MasterProj/Library/metadata/49/49d187ecef3e8a8478dba4c373bed2db.info
	MasterProj/Library/metadata/49/49f15f5796f2e524c83743cc71377c8a
	MasterProj/Library/metadata/49/49f15f5796f2e524c83743cc71377c8a.info
	MasterProj/Library/metadata/4a/4a5a417da7040b141859fd4fd142352b
	MasterProj/Library/metadata/4a/4a5a417da7040b141859fd4fd142352b.info
	MasterProj/Library/metadata/4a/4afa6f93b6209f240bc4402adffab007
	MasterProj/Library/metadata/4a/4afa6f93b6209f240bc4402adffab007.info
	MasterProj/Library/metadata/4d/4dcf6ba4829e2fd49a0af5a5f898a7d8.info
	MasterProj/Library/metadata/4d/4dfd3f5da88944ec6ac5977d676c30c6
	MasterProj/Library/metadata/4e/4e256d1c5d35c774ea2778bc1b6512ef
	MasterProj/Library/metadata/4e/4e256d1c5d35c774ea2778bc1b6512ef.info
	MasterProj/Library/metadata/4e/4eb2d9b39731b4e43a77cc0c01753428
	MasterProj/Library/metadata/4e/4eb2d9b39731b4e43a77cc0c01753428.info
	MasterProj/Library/metadata/4f/4f768b64e6c37495699fffe3007e4379
	MasterProj/Library/metadata/51/51c00e240d423b4499b1ab212533258d.info
	MasterProj/Library/metadata/52/52f625a1ed593324a8654ccf97e5630d
	MasterProj/Library/metadata/52/52f625a1ed593324a8654ccf97e5630d.info
	MasterProj/Library/metadata/54/5489cca054e631545914525d169e4f6e.info
	MasterProj/Library/metadata/54/54c2b671ed1e1674e9de0c3744dd3f53
	MasterProj/Library/metadata/54/54c2b671ed1e1674e9de0c3744dd3f53.info
	MasterProj/Library/metadata/57/57b0cb4ca0352684480792692c674215
	MasterProj/Library/metadata/57/57b0cb4ca0352684480792692c674215.info
	MasterProj/Library/metadata/59/595b15a09ee3ed44e8b91d861fb4aca7
	MasterProj/Library/metadata/59/595b15a09ee3ed44e8b91d861fb4aca7.info
	MasterProj/Library/metadata/5e/5e88f2fe889d701418a46e4013d43509
	MasterProj/Library/metadata/5e/5e88f2fe889d701418a46e4013d43509.info
	MasterProj/Library/metadata/62/62a263faffd6bc343945062f2fb8ddb5
	MasterProj/Library/metadata/62/62a263faffd6bc343945062f2fb8ddb5.info
	MasterProj/Library/metadata/63/63b33760e71c1d648b8d5f4749a4d0bc
	MasterProj/Library/metadata/63/63b33760e71c1d648b8d5f4749a4d0bc.info
	MasterProj/Library/metadata/66/66003687e82168b4cb7e7fbc9dff2667
	MasterProj/Library/metadata/66/66003687e82168b4cb7e7fbc9dff2667.info
	MasterProj/Library/metadata/68/68032108798135849b3ba0284d516e81
	MasterProj/Library/metadata/68/68032108798135849b3ba0284d516e81.info
	MasterProj/Library/metadata/69/694a3a698d576d940bd8c5e79cf8ccac.info
	MasterProj/Library/metadata/6a/6ae9ff7bfa6120641806cd59f4692992.info
	MasterProj/Library/metadata/6c/6cc425f914db354479dc05c116201fde
	MasterProj/Library/metadata/6c/6cc425f914db354479dc05c116201fde.info
	MasterProj/Library/metadata/6d/6db685cd656a0b048bf0a2ed23851d08
	MasterProj/Library/metadata/6d/6db685cd656a0b048bf0a2ed23851d08.info
	MasterProj/Library/metadata/6e/6e582440ac03b46459d60b6ad731717e
	MasterProj/Library/metadata/6e/6e582440ac03b46459d60b6ad731717e.info
	MasterProj/Library/metadata/6f/6f4300965951e2e47ab83c567f9183e7
	MasterProj/Library/metadata/6f/6f4300965951e2e47ab83c567f9183e7.info
	MasterProj/Library/metadata/75/75a31d4cc3c4e504fbfb30a87dee3548
	MasterProj/Library/metadata/75/75a31d4cc3c4e504fbfb30a87dee3548.info
	MasterProj/Library/metadata/75/75debb50ca23b6c46bf810b0843a9dd9
	MasterProj/Library/metadata/75/75debb50ca23b6c46bf810b0843a9dd9.info
	MasterProj/Library/metadata/76/76cb1fb04a0d9454d9961fb6b6f03ee4
	MasterProj/Library/metadata/76/76cb1fb04a0d9454d9961fb6b6f03ee4.info
	MasterProj/Library/metadata/79/791b9c53a0f10e74bb2fe793d6f7cb31
	MasterProj/Library/metadata/79/791b9c53a0f10e74bb2fe793d6f7cb31.info
	MasterProj/Library/metadata/79/799b9b43d55071549abd0e03fd6f5e77
	MasterProj/Library/metadata/79/799b9b43d55071549abd0e03fd6f5e77.info
	MasterProj/Library/metadata/7a/7ac13a132a555cd4a871b38d93a48c80.info
	MasterProj/Library/metadata/7c/7c67ec16dcb540f4997bb4396311fc56
	MasterProj/Library/metadata/7c/7c67ec16dcb540f4997bb4396311fc56.info
	MasterProj/Library/metadata/7f/7f148f7708152124dbb88535e6dc4949
	MasterProj/Library/metadata/7f/7f148f7708152124dbb88535e6dc4949.info
	MasterProj/Library/metadata/80/8017b6c14d4704248b210b5803d00eb0
	MasterProj/Library/metadata/80/8017b6c14d4704248b210b5803d00eb0.info
	MasterProj/Library/metadata/80/80ae532b0b954ec45ab2db13db15528b
	MasterProj/Library/metadata/80/80ae532b0b954ec45ab2db13db15528b.info
	MasterProj/Library/metadata/81/811611bb2a989b0468ce2eda72a0941e
	MasterProj/Library/metadata/81/811611bb2a989b0468ce2eda72a0941e.info
	MasterProj/Library/metadata/81/81c13b204cccc254fafcaa5bb758ad09
	MasterProj/Library/metadata/81/81c13b204cccc254fafcaa5bb758ad09.info
	MasterProj/Library/metadata/82/82437e9b28c15d340b73be53764b3850.info
	MasterProj/Library/metadata/82/82784c7121fe2df429e27e2dcb9c745c
	MasterProj/Library/metadata/82/82784c7121fe2df429e27e2dcb9c745c.info
	MasterProj/Library/metadata/85/8561b9643090ff44b982878e839b110b
	MasterProj/Library/metadata/85/8561b9643090ff44b982878e839b110b.info
	MasterProj/Library/metadata/86/8660dd9cef3f801468f3fe33f718e5f1
	MasterProj/Library/metadata/86/8660dd9cef3f801468f3fe33f718e5f1.info
	MasterProj/Library/metadata/88/88ec7c192e88ae14f97b2f47443550c9
	MasterProj/Library/metadata/88/88ec7c192e88ae14f97b2f47443550c9.info
	MasterProj/Library/metadata/8b/8bb59ff79022a85429c50e4469c52f9d
	MasterProj/Library/metadata/8b/8bb59ff79022a85429c50e4469c52f9d.info
	MasterProj/Library/metadata/8d/8d86309e6c799c94797b13448c74fc5a
	MasterProj/Library/metadata/8d/8d86309e6c799c94797b13448c74fc5a.info
	MasterProj/Library/metadata/8f/8fbfbf80b0414404e981bcd482f251cb
	MasterProj/Library/metadata/8f/8fbfbf80b0414404e981bcd482f251cb.info
	MasterProj/Library/metadata/91/913aad696e6a1b14aa100e08b1d138ff
	MasterProj/Library/metadata/91/913aad696e6a1b14aa100e08b1d138ff.info
	MasterProj/Library/metadata/91/9155cba92c1fee44093f504fb98afd0e
	MasterProj/Library/metadata/91/9155cba92c1fee44093f504fb98afd0e.info
	MasterProj/Library/metadata/92/92763eb9695f7a24cabbcfa558997f89
	MasterProj/Library/metadata/92/92763eb9695f7a24cabbcfa558997f89.info
	MasterProj/Library/metadata/93/93c1370867fcb45079592da39e0994c5
	MasterProj/Library/metadata/98/987f23fc35a973549af10bb28f533478
	MasterProj/Library/metadata/98/987f23fc35a973549af10bb28f533478.info
	MasterProj/Library/metadata/9e/9e8b6df2f19927640b09aa71886c9942
	MasterProj/Library/metadata/9e/9e8b6df2f19927640b09aa71886c9942.info
	MasterProj/Library/metadata/a0/a07a794346a402b4ea5e5e7a4af510f9
	MasterProj/Library/metadata/a0/a07a794346a402b4ea5e5e7a4af510f9.info
	MasterProj/Library/metadata/a5/a50961172e5dc89448b1fbf9d121dece
	MasterProj/Library/metadata/a5/a50961172e5dc89448b1fbf9d121dece.info
	MasterProj/Library/metadata/a8/a8d9e44eff3f447db9448925617ccb44
	MasterProj/Library/metadata/aa/aa268354da1713e44a784437c5a609b9
	MasterProj/Library/metadata/aa/aa268354da1713e44a784437c5a609b9.info
	MasterProj/Library/metadata/ab/abdc053b8e3644f299aedc641d609a84
	MasterProj/Library/metadata/ac/ac516f1dc60e02042acbe6a62f517b2e
	MasterProj/Library/metadata/ac/ac516f1dc60e02042acbe6a62f517b2e.info
	MasterProj/Library/metadata/ac/ac7a821d37bc32f4ab866c93272aac11
	MasterProj/Library/metadata/ac/ac7a821d37bc32f4ab866c93272aac11.info
	MasterProj/Library/metadata/ad/ad230fd9ae7375c46b1b3ac48ad1e4e0
	MasterProj/Library/metadata/ad/ad230fd9ae7375c46b1b3ac48ad1e4e0.info
	MasterProj/Library/metadata/ae/aefcecd1318cbb042bd8bf336f309e4a.info
	MasterProj/Library/metadata/b4/b4ffd486f6cbf2645957f99e338a1d9a
	MasterProj/Library/metadata/b4/b4ffd486f6cbf2645957f99e338a1d9a.info
	MasterProj/Library/metadata/b8/b8a455baeb4f08840a2cc49bdf059eed
	MasterProj/Library/metadata/b8/b8a455baeb4f08840a2cc49bdf059eed.info
	MasterProj/Library/metadata/b9/b902e7b6f02497c46b780131688455ff
	MasterProj/Library/metadata/b9/b902e7b6f02497c46b780131688455ff.info
	MasterProj/Library/metadata/b9/b94e5481ba859b340bdd6c878bd29cc7
	MasterProj/Library/metadata/b9/b94e5481ba859b340bdd6c878bd29cc7.info
	MasterProj/Library/metadata/ba/ba6b12f4b7e1eed4b8bcb3e568ceda44
	MasterProj/Library/metadata/ba/ba6b12f4b7e1eed4b8bcb3e568ceda44.info
	MasterProj/Library/metadata/bd/bdb9d1dd5fc134b46aeec6e8efe8a93c
	MasterProj/Library/metadata/bd/bdb9d1dd5fc134b46aeec6e8efe8a93c.info
	MasterProj/Library/metadata/c4/c452c88a87f3d48ae945e7bcf1b4a6ba
	MasterProj/Library/metadata/c6/c60cdcba55c3b83498cb1fe03f96e4dc.info
	MasterProj/Library/metadata/c7/c73e4b467514d5c43a671047564a2019.info
	MasterProj/Library/metadata/c7/c7559cdbc33f04af1a5a42d2aa5b40a5
	MasterProj/Library/metadata/c7/c7e641746d7e9c94396d43cc38d90d83
	MasterProj/Library/metadata/c7/c7e641746d7e9c94396d43cc38d90d83.info
	MasterProj/Library/metadata/c9/c9593917fb9d2684db392cd72a2ab238
	MasterProj/Library/metadata/c9/c9593917fb9d2684db392cd72a2ab238.info
	MasterProj/Library/metadata/ca/ca8fd7dff61c5ff45b7cf3288b76b42d.info
	MasterProj/Library/metadata/ca/ca9290669bcf49e4f95fa0d16efc3f4e.info
	MasterProj/Library/metadata/cb/cb575fd0556214247a32e97a889a66ee
	MasterProj/Library/metadata/cb/cb575fd0556214247a32e97a889a66ee.info
	MasterProj/Library/metadata/cb/cbe45637d8a224244a303315a6ad44f2
	MasterProj/Library/metadata/cb/cbe45637d8a224244a303315a6ad44f2.info
	MasterProj/Library/metadata/cb/cbf292ab5ecce8847a36e0efea2ee60e
	MasterProj/Library/metadata/cb/cbf292ab5ecce8847a36e0efea2ee60e.info
	MasterProj/Library/metadata/cc/cccde4c5ffb739943ba6857307a20901
	MasterProj/Library/metadata/d1/d13b214e05011364d8ef13c8165365c3
	MasterProj/Library/metadata/d1/d13b214e05011364d8ef13c8165365c3.info
	MasterProj/Library/metadata/d4/d46c98294e8d5bd439b56bdd5cfcbbc4
	MasterProj/Library/metadata/d4/d46c98294e8d5bd439b56bdd5cfcbbc4.info
	MasterProj/Library/metadata/d4/d4e3f84596c7f5440af5e4eca66163dc
	MasterProj/Library/metadata/d4/d4e3f84596c7f5440af5e4eca66163dc.info
	MasterProj/Library/metadata/d8/d831ed733ccc5fc40a9818ccee0bfa73
	MasterProj/Library/metadata/d8/d831ed733ccc5fc40a9818ccee0bfa73.info
	MasterProj/Library/metadata/da/dae499942b6ae954287e421f05bb7cc9
	MasterProj/Library/metadata/da/dae499942b6ae954287e421f05bb7cc9.info
	MasterProj/Library/metadata/db/db4a6ce3afb7e734e9299f4bb6646d60
	MasterProj/Library/metadata/db/db4a6ce3afb7e734e9299f4bb6646d60.info
	MasterProj/Library/metadata/e1/e19df042f88d40a479332d7694af7680.info
	MasterProj/Library/metadata/e2/e27fbc2a2ad024340bf5149a62a022fc
	MasterProj/Library/metadata/e2/e27fbc2a2ad024340bf5149a62a022fc.info
	MasterProj/Library/metadata/e2/e2b0c9e129989f94da98fdd68e64ddae
	MasterProj/Library/metadata/e2/e2b0c9e129989f94da98fdd68e64ddae.info
	MasterProj/Library/metadata/e3/e333ab3bcb187d04ea56007fe778181e
	MasterProj/Library/metadata/e3/e333ab3bcb187d04ea56007fe778181e.info
	MasterProj/Library/metadata/e5/e55f54c46cc3d844998d0d6c1a6e7dcc
	MasterProj/Library/metadata/e5/e55f54c46cc3d844998d0d6c1a6e7dcc.info
	MasterProj/Library/metadata/e8/e86f761fb8428ce4d98a7d26667c1d7e
	MasterProj/Library/metadata/e8/e86f761fb8428ce4d98a7d26667c1d7e.info
	MasterProj/Library/metadata/ea/ea5c4926ec4052d4ab4e51ab4ec59fe7
	MasterProj/Library/metadata/ea/ea5c4926ec4052d4ab4e51ab4ec59fe7.info
	MasterProj/Library/metadata/ea/ead9f8d8ce0a06049aac502886226b46
	MasterProj/Library/metadata/ea/ead9f8d8ce0a06049aac502886226b46.info
	MasterProj/Library/metadata/eb/ebf9cd8ddf5ad9745b275a541df455ba.info
	MasterProj/Library/metadata/ec/ec13f1e360611844fb7e33e244101587.info
	MasterProj/Library/metadata/f3/f306914e412146146af1eb2f431c3776.info
	MasterProj/Library/metadata/f4/f4da9cbb408be44968e85ee20767deef
	MasterProj/Library/metadata/f4/f4ee874d8acd26543b7c742242f771d0.info
	MasterProj/Library/metadata/f5/f5b89ea417ae34244851660f1d5017cc.info
	MasterProj/Library/metadata/f8/f8eca3a711f4842eba0c52ea9885ef7f
	MasterProj/Library/shadercompiler-UnityShaderCompiler.exe0.log

Commit:
bhio

Commit:
Combined improved Keyboard code with Controller Code

Based on yifan --> master merge.

YifanXu reviewed (approved):
Raghav said the controller section is working so

Commit:
Changed Test Sprites

Somebody says they hurt their eyes. So I took sprites off of the Internet that reflect what the test enemies should do. The player sprite is a robot.
